### PR TITLE
perf: update sso tab template to use latest sdk and mgt

### PR DESCRIPTION
**Pull request by SLdragon**
need to bypass lint for a better tsx, jsx file

### CODE REVIEW

File: templates/scenarios/js/sso-tab/src/components/sample/PersonCardGraphToolkit.jsx [hunk 1]
```diff
@@ -7,23 +7,37 @@ export function PersonCardGraphToolkit(props) {
...
+        <a
+          href="https://docs.microsoft.com/en-us/graph/toolkit/components/person-card"
+          target="_blank"
+          rel="noreferrer"
+        >
+          person card component
+        </a>{" "}
+        with&nbsp;
+        <a
+          href="https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/next/teamsfx/packages/providers/mgt-teamsfx-provider"
```
Comment (yiqing-zhao) [inline templates/scenarios/js/sso-tab/src/components/sample/PersonCardGraphToolkit.jsx:21]: The link is linked to `next/teamsfx` branch, should we use the latest main branch instead?

File: templates/scenarios/js/sso-tab/src/components/sample/AzureFunctions.jsx [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-import { BearerTokenAuthProvider, createApiClient } from "@microsoft/teamsfx"
+import { BearerTokenAuthProvider, createApiClient } from "@microsoft/teamsfx";
 import { TeamsFxContext } from "../Context";
 import config from "./lib/config";
 
 const functionName = config.apiName || "myFunc";
 
-async function callFunction(teamsfx) {
-  if (!teamsfx) {
+async function callFunction(credential) {
```
Comment (hund030) [inline templates/scenarios/js/sso-tab/src/components/sample/AzureFunctions.jsx:10]: How about using naming convention `teamsUserCredential` to keep consistency?

Comment (SLdragon) [reply templates/scenarios/js/sso-tab/src/components/sample/AzureFunctions.jsx:10]: Sure, thanks

File: templates/scenarios/js/sso-tab/src/components/sample/PersonCardGraphToolkit.jsx (hunk 1 above)
Comment (SLdragon) [reply templates/scenarios/js/sso-tab/src/components/sample/PersonCardGraphToolkit.jsx:21]: Sure, thanks

Comment (SLdragon) [reply templates/scenarios/js/sso-tab/src/components/sample/PersonCardGraphToolkit.jsx]: fixed

File: templates/scenarios/js/sso-tab/src/components/sample/AzureFunctions.jsx (hunk 2 above)
Comment (SLdragon) [reply templates/scenarios/js/sso-tab/src/components/sample/AzureFunctions.jsx]: fixed

**Review by hund030 — APPROVED** [review]

**Review by yiqing-zhao — APPROVED** [review]

**Pull request merged**